std_msgs/Header header
geometry_msgs/Pose pose
sensor_msgs/PointCloud2 obstacles